addons/account/models/account_move.py

```diff
@@ -1923,8 +1923,7 @@ def _get_unbalanced_moves(self, container):
                    ROUND(SUM(line.credit), currency.decimal_places) credit
               FROM account_move_line line
               JOIN account_move move ON move.id = line.move_id
-              JOIN res_company company ON company.id = move.company_id
-              JOIN res_currency currency ON currency.id = company.currency_id
+              JOIN res_currency currency ON currency.id = move.currency_id
```

Comment:
This is completely wrong. debit/credit/balance are expressed in *company's currency*. I don't know what you are trying to fix but the problem is probably on the pos closing entry, not here.